iceoryx_binding_c/include/iceoryx_binding_c/subscriber.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IOX_BINDING_C_SUBSCRIBER_H_
+#define IOX_BINDING_C_SUBSCRIBER_H_
+
+#include "iceoryx_binding_c/internal/c2cpp_bridge.h"
+#include "iceoryx_binding_c/types.h"
+
+CLASS SubscriberPortData*
+Subscriber_new(const char* const service, const char* const instance, const char* const event, uint64_t historyRequest);
```

Comment:
runnable name missing

Reply:
Same argument as before with the Publisher. 2 Options to create a subscriber, without runnable handle directly into the default runnable otherwise into the provided runnable.